hexrd/ui/image_mode_widget.py

```diff
@@ -408,6 +415,17 @@ def on_eta_max_changed(self, max_value):
                 HexrdConfig().set_polar_res_eta_min(min_value, rerender=False)
         HexrdConfig().polar_res_eta_max = max_value
 
+    def show_overlay_manager(self):
+        if hasattr(self, '_overlay_manager'):
+            self._overlay_manager.ui.reject()
+            del self._overlay_manager
+
+        self._overlay_manager = AzimuthalOverlayManager(self.ui)
+        self._overlay_manager.show()
```

Comment:
Can we change all of the `overlay_manager` names to `polar_overlay_manager`? This is just to make it clear in the code that this is in the polar view, since the image mode widget has in it logic for all of the different views.

Reply:
Done.